pkgs/gcc/gcc.yaml

```diff
@@ -12,6 +12,19 @@ sources:
   url: http://ftpmirror.gnu.org/gcc/gcc-4.9.2/gcc-4.9.2.tar.bz2
 
 build_stages:
+  - name: rpath_patch
+    before: rpath_sed
+    files: [rpath.patch]
+    handler: bash
+    bash: |
+      patch -up1 < _hashdist/rpath.patch
+
+  - name: rpath_sed
+    before: configure
+    handler: bash
+    bash: |
+      sed -i "s|@@ARTIFACT@@|${ARTIFACT}|g" gcc/config/i386/gnu-user.h gcc/config/i386/gnu-user64.h
```

Comment:
I think these two blobs can just be merged into one. I'll do it after my gcc build finishes.